ui/components/section/section.go

```diff
@@ -52,6 +56,23 @@ type NewSectionOptions struct {
 	CreatedAt   time.Time
 }
 
+func (options NewSectionOptions) GetConfigFiltersWithCurrentRemoteAdded(ctx *context.ProgramContext) string {
+	searchValue := options.Config.Filters
+	if !ctx.Config.SmartFilteringAtLaunch {
+		return searchValue;
+	}
+	repo, err := repository.Current()
+	if err != nil {
+		return searchValue
+	}
+	for _, token := range strings.Fields(searchValue) {
+		if strings.HasPrefix(token, "repo:") {
+			return searchValue
+		}
+	}
+	return fmt.Sprintf("%s repo:%s/%s", searchValue, repo.Owner, repo.Name)
```

Comment:
Let's actually put the `repo:%s/%s` first so user can clearly see we've added it